### PR TITLE
Show cause of bug for antranig and alfredo

### PR DESCRIPTION
**Pull request by kaspermarkus**

### CODE REVIEW

**Comment by colinbdclark** [comment]
A description of the issues related to this pull request would be very helpful for those of us who are not quite fully in the loop, but trying to follow along supportively. :cat2:

File: lib/dataSource.js [hunk 1]
```diff
@@ -321,7 +322,8 @@ kettle.dataSource.URL.handle.http = function (path, options, data) {
             host: urlObj.hostname,
             port: parseInt(urlObj.port, 10),
             path: urlObj.path,
-            method: "GET"
+            method: "GET",
+            protocol: urlObj.protocol
```
Comment (amb26) [inline lib/dataSource.js]: Assigned to a variable but never used - I assume this doesn't pass linting

Comment (amb26) [inline lib/dataSource.js]: Ah sorry, ignore this comment - I didn't see the diff context

**Comment by kaspermarkus** [comment]
Yeah, good point, @colinbdclark !

Actually this was never meant to be a pull - I just meant to do a diff :)

The issue is our broken production tests in the GPII trunk that alfredo found when working on the CI. I triaged them and found that the issue stems from the fact that we introduced https on the preferences server (with a 301 redirect when using HTTP). It turns out that Kettle doesnt support https - the fix in this patch causes the tests to succeed. It's worth noting as well that an alternative to this solution when using node 4.x.x is to simply pass the 'protocol' as one of the requestOptions to node's http.request. This is not supported in older versions of node

(see also https://botbot.me/freenode/fluid-work/2016-05-12/?tz=Europe/Berlin)

**Pull request closed without merging**